gnd/src/main/java/com/google/android/gnd/persistence/local/room/entity/FeatureMutationEntity.java

```diff
@@ -53,36 +53,14 @@
     indices = {@Index("feature_id")})
 public abstract class FeatureMutationEntity extends MutationEntity {
 
-  @CopyAnnotations
-  @NonNull
-  @ColumnInfo(name = "feature_id")
-  public abstract String getFeatureId();
-
-  @CopyAnnotations
-  @NonNull
-  @ColumnInfo(name = "layer_id")
-  public abstract String getLayerId();
-
-  /** Non-null if the feature's location was updated, null if unchanged. */
-  @CopyAnnotations
-  @Nullable
-  @Embedded
-  public abstract Coordinates getNewLocation();
-
-  /** Non-empty if a polygon's vertices were updated, null if unchanged. */
-  @CopyAnnotations
-  @Nullable
-  @ColumnInfo(name = "polygon_vertices")
-  public abstract String getNewPolygonVertices();
```

Comment:
Could you please revert these unrelated code refactors to reduce the noise? Also applicable everywhere else. Thanks!

Reply:
good catch, not even sure why this stuff was moved

Reply:
done, for some reason when I run the formatter now it wants to move all this stuff -- could it have something to do with kotlin settings?